[DOCS-9481] Update Investigate Sensitive Data Issues (#28146)
DataDog/documentation: content/en/sensitive_data_scanner/guide/investigate_sensitive_data_issues.md

- Use [Audit Trail][3] to see who may have accessed this sensitive data within Datadog, **View in Audit Trail** in the **Users who accessed these events** section.

{{< img src="sensitive_data_scanner/investigate_sensitive_data_issues/case_mgmt_02_01_2024.png" alt="The case page showing information about the security issue, the assignee and creator of the case, and a timeline of events" style="width:60%;">}}

[1]: /service_management/case_management/
[2]: /service_management/incident_management/
[3]: /account_management/audit_trail

{{% /tab %}}
{{% tab "Cloud Storage" %}}

Click the **Datastores with Sensitive Data** tab to see all sensitive data issues for Cloud Storage.

In the **xxx Datastores with Sensitive section**, click on any of the dropdown menus to filter on datastores based on the type of sensitive data, account, region, team and so on.

To investigate a datastore:

1. Click on a datastore.
1. You can view files where sensitive data was found and then click on a file to inspect it in AWS.
  Datadog recommends doing the following:
    - Review a few files to get a sense of the classification accuracy.

>     - Follow up with the team or service owner (shown in the side panel) to confirm if sensitive data is supposed to be in that bucket.

Reviewer: ```suggestion
    - Follow up with the team or service owner listed in the side panel to confirm whether sensitive data is meant to be in the bucket.
```

>       - If it is not supposed to be in that bucket, delete the files or move them to an appropriate bucket.

Reviewer: ```suggestion
      - If it is not supposed to be in the bucket, delete the files or move them to an appropriate bucket.
```

>       - If it is supposed to be in that bucket, follow the following steps to improve your security posture:

Reviewer: ```suggestion
      - If it is supposed to be in that bucket, complete the following steps to improve your security posture:
```

Author: This was a duplicate suggestion. Committed the other one.

>       - If it is supposed to be in that bucket, follow the following steps to improve your security posture:

Reviewer: ```suggestion
      - If it is supposed to be in the bucket, complete the following steps to improve your security posture:
```

        1. Click the **Security** tab in the side panel and review the **Misconfigurations** section.
        1. Click on a misconfiguration to see details in Cloud Security Management.

>         1. In the **Next Steps** section:

Reviewer: I think the steps that follow this one may not be indented correctly.

Author: Thanks! Fixed it.

          1. Under **Triage**, click the dropdown to change the triage status of the signal. The default status is `OPEN`.
          1. Click **Assign Signal** to assign a signal to yourself or another Datadog user.
          1. Click **See remediation** to see more information on how to remediate the issue.
          1. Under **More Actions**, you can add a Jira issue, run workflows, or add a comment.

>         To run a workflow, select **Run Workflow** and then in the workflow browser, search and select a workflow to run. See Automate Security Workflows with Workflow Automation for more information.

Reviewer: It looks like the link is missing here.

Author: Added it
